contracts/test/proposals/vips/vip16.sol

```diff
@@ -635,6 +676,12 @@ contract vip16 is Proposal {
             address(core)
         );
 
+        assertApproxEq(
+            oracle.getCurrentOraclePrice().toInt256(),
+            oldOracle.getCurrentOraclePrice().toInt256(),
+            0
+        );
```

Comment:
that's true in the validate step, but after next block, APR is different 👍

Reply:
This check will pass on mainnet because it's asserting the oracle price is approximately equal under 1 basis point of difference. I would do an exact check, but there are other things that happen between the oracle getting initialized and the validation step so the prices won't be exactly the same.